src/signals/incident-management/components/Label/__test__/Label.test.js

```diff
@@ -12,8 +12,8 @@ describe.skip('signals/incident-management/components/Label', () => {
     const { container, getByLabelText } = render(
       withAppContext(
         <div>
-          <Label htmlFor="someOtherElementId">This is my label text</Label>
-          <input type="text" id="someOtherElementId" />
+          <Label htmlFor="someOtherElementId" >This is my label text</Label>
+          <input type="text" id="someOtherElementId" isGroupHeader={false} />
```

Comment:
`isGroupHeader` can be left out here; isn't a valid attribute for `<input>`

Reply:
gedaan